pages/index.vue

```diff
@@ -98,74 +101,123 @@
             </button>
           </span>
         </li>
-      </ul>
-    </transition>
+      </transition-group>
+    </ul>
   </main>
 </template>
 
 <script lang>
 import { defineComponent } from '@nuxtjs/composition-api'
 import addTask from '~/components/addTask.vue'
+import axios from 'axios'
+import Search from '~/components/search.vue';
+const API_BASE_URL = 'https://todo-app-csoc.herokuapp.com/';
```

Comment:
You can also create a configuration for the default request header since the base URL is being used all over the project.

https://axios-http.com/docs/config_defaults